### PR TITLE
Feature/add tests docs and gh actions

### PR DESCRIPTION
**Pull request by ERuban**
- add tests
- add docs
- add github CI action

### CODE REVIEW

File: docs/ecs/sniffs.md [hunk 1]
```diff
@@ -0,0 +1,523 @@
+---eonx_docs---
+title: 'EasyStandard: Sniffs'
+weight: 1001
+---eonx_docs---
+
+#### Arrays
+
+##### [\EonX\EasyStandard\Sniffs\Arrays\AlphabeticallySortedArrayKeysSniff][1]
```
Comment (roman-eonx) [inline docs/ecs/sniffs.md:8]: Please fix all the namespaces like this.

File: docs/rectors/settings.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
+---eonx_docs--- title: Rector settings weight: 2000 is_section: true ---eonx_docs---
+
+### Example configuration
+
+```php
+// rector.php
+declare(strict_types=1);
+
+use EonX\EasyStandard\Rector\StrictInArrayRector;
```
Comment (roman-eonx) [inline docs/rectors/settings.md:9]: Here as well.

**Review by roman-eonx — CHANGES_REQUESTED** [review]

File: docs/ecs/sniffs.md (hunk 1 above)
Comment (ERuban) [reply docs/ecs/sniffs.md]: Done

File: docs/rectors/settings.md (hunk 2 above)
Comment (ERuban) [reply docs/rectors/settings.md]: Done

**Review by roman-eonx — APPROVED** [review]

**Pull request merged**